com.wire.WireDesktop.appdata.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
By "contributed upstream" you meant to pull request this file `com.wire.WireDesktop.appdata.xml` into this repository `https://github.com/wireapp/wire-desktop/`?

If so, when (and if) that happens, where should that be located?

The reason I'm asking is because they use `electron-builder` to build the application. If that tool is not auto-generating this file already, then they would have to do something on their own to update the `<release ... />` tag(s), right?

I guess I'm not sure how that works in general :sweat_smile: ...for me creating a pull request there is going to be easy, but explaining them the reasons it might be tricky. Anyway, I'll do it, maybe they know what to do with this file and how to update the `<release ... />` tag(s), which is the only part I see that changes (frequently) between releases.

Reply:
This is done via wireapp/wire-desktop/pull/1785 ...not sure if it's entirely correct though.